Pocket/recommendation-api
# Remove nonfunctional getTopicRecommendations query

# Goal
Cleanup the schema before upgrading the graphql library in https://github.com/Pocket/recommendation-api/pull/945:

1. Remove unused `getTopicRecommendations` query. This query has not been working for a year, because I removed the code behind this endpoint a year ago in https://github.com/Pocket/recommendation-api/pull/409, but forgot to remove it from the schema.
2. Fix typos in descriptions.
3. Remove `RankedCorpusSlate` and `RankedCorpusSlateExperiment` types. The `getRankedCorpusSlate` query for which these were used was removed in https://github.com/Pocket/recommendation-api/pull/947.

# QA
The Apollo check passes, so there are no breaking changes. If there were any breaking changes in this PR, this check would've failed.
